zuuler/fertiger.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env python3
+#
+# Copyright 2021 Red Hat, Inc.
+# All Rights Reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+#
+
+from collections import defaultdict
+import os.path
+from pathlib import Path
+
+from jinja2 import Template
+
+
+JOBS_TO_COLLECT = [
```

Comment:
Do we expect JOB_TO_COLLECT and JOBS_REMAPPING to have different jobs? if not, then JOB_TO_COLLECT.keys() can be a good excuse to just drop JOBS_TO_COLLECT completely.

Reply:
That's to be defined actually. From my understanding, we may want to keep only some of the defined jobs (JOBS_TO_COLLECT) and for some we may want to update their names (JOBS_REMAPPING). I can rewrite that into one dict variable if that's clear for you: if key is in dict, collect the job, and if there is value for given key (other that None), also change its name. Does it make sense?